src/api/api.c

```diff
@@ -227,6 +231,8 @@ z_bytes_t z_query_parameters(const z_query_t *query) {
     return parameters;
 }
 
+z_value_t z_query_payload_value(const z_query_t *query) { return query->_with_value; }
```

Comment:
What about just calling it `z_query_payload`? 
Or `z_query_body`?

Reply:
`z_query_value` was agreed.